Twig/RollbarExtension.php

```diff
@@ -89,6 +89,11 @@ public function getInitRollbarCode(array $context)
             );
         }
 
+        $rollbarJsVersion = 'v1';
```

Comment:
then probably we will be forced to change all embedding code, so it does not matter :)